### ISSUE TITLE
[FEATURE] README to be created for the website project

### ISSUE THREAD

**Issue by yashsehgal** [DesignSystemsOSS/thedesignsystems#27]
### ⭐ Suggestion

We need to create a proper README.md file to explain all the necessary steps and descriptions of this project.
Such as About project, Tech Stack section, How to set up the local-dev environment, and other helpful links/resources.

### 💻 Use Cases

This will help the contributors to get started with ease and understand the project workflow in a better way.

### ❌ Related Problems

Nothing as such, It's a completely new feature

**Comment by danger-ahead** [comment]
Can I work on this?

**Comment by yashsehgal** [comment]
Yes @danger-ahead You can get started with this issue. Make sure you're creating a new unique branch from the `development` branch and raise a PR to `development` only. Go ahead with this issue and feel free to ask for queries. 🚀

**Comment by danger-ahead** [comment]
okay. please consider assigning me.
also please provide me with the necessary details to be added
